lib/application.js

```diff
@@ -150,12 +150,8 @@ Application.prototype.createClient = function () {
       args.push(arg)
     })
 
-    var launcherPath = null
-    if (process.platform === 'win32') {
-      launcherPath = path.join(__dirname, '..', 'bin', 'launcher.exe')
-    } else {
-      launcherPath = path.join(__dirname, 'launcher.js')
-    }
+    const isWin = (process.platform === 'win32')
```

Comment:
I'd prefer not to use `const` in spectron yet to not break any old node versions that people may still be using.